libexec/slurm/monitor.sh

```diff
@@ -49,21 +49,40 @@ read -r -d '' template <<'TEMPLATE' || true
 TEMPLATE
 
 # Fetch the state of the job
-control=$(scontrol show job "$1" --oneline | head -n 1 | tr ' ' '\n' 2>&1)
+raw_control=$(scontrol show job "$1" --oneline 2>&1)
 exit_status="$?"
+control=$(echo "$raw_control" | head -n 1 | tr ' ' '\n')
+cat <<EOF
+scontrol:
+$control
+EOF
```

Comment:
Is this debugging code that should be removed?

Reply:
ping

Reply:
https://github.com/openflighthpc/flight-job/pull/14/commits/8dbac91b8f831737f4e38e773d163dac7a27bbc5

Reply:
https://github.com/openflighthpc/flight-job/pull/14/commits/bd68dd00f25388794df831dc5bc8e97ca9087545